packages/blade/src/components/ActionList/ActionList.tsx

```diff
@@ -28,7 +29,7 @@ type ActionListProps = {
   /**
    * Decides the backgroundColor of ActionList
    */
-  surfaceLevel?: 2 | 3;
+  surfaceLevel?: Exclude<SurfaceLevels, 1>;
```

Comment:
han for same reason I added this because adding 2,3 is too much of an effort everytime and if 4, 5 comes in we'll anyways expose them its just that we want to skip level1 card on level1 surface.

Reply:
Then fine.